source/getting_started/aiot_intelligent_edge/coco_for_applications/end_user_client_applications/android_end_user_client_applications/pre_install_checklist_android_end_user_client_apps.rst

```diff
@@ -5,7 +5,21 @@
 Pre-Install Checklist
 =====================
 
-.. figure:: /_static/coming-soon.png
-   :align: center
+The following are necessary to try out the **COCO IoT SDK Android** and **IoT Sample App**:
 
-   **Coming Soon**
+Git
+    - To clone the Sample App repo from Github.
+    - `Git Installation <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+
+Android Studio
+    - To run the Sample App on an Android device or an emulator.
+    - `Android Studio Installation <https://developer.android.com/studio/install>`_
+
+Generating Client ID and Redirect URI
+    - The IoT example app shows a login solution for developers to get apps authorized by a COCO user and build value-added service extensions for the COCO platform.
+    - Therefore *Client ID* and *Redirect URI* are essential for a developer to handle authentication of his user. The steps for generation are found in :ref:`Manage the Buzz <native_apps>` section.
```

Comment:
Change the link caption from 'Manage the Buzz' to 'Native Apps'

Reply:
changed.